src/pages/Homepage.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <SRNEntry
+    :redirectTo="redirectTo"
+    :redirectID="redirectID"
+    :purpose="purpose"
+    :subPurpose="subPurpose"
+    :plioUse="plioUse"
+  />
+  <div v-if="!plioUse">
+    <h3 class="mx-auto">OR</h3>
+    <StudentSelect />
+  </div>
+</template>
+
+<script>
+import StudentSelect from "@/components/StudentSelect.vue";
+import SRNEntry from "@/components/SRNEntry.vue";
+
+/* Have a variable plioUse that will help toggle between divs, one that will disable the entry of more 
```

Comment:
Hmm... I don't see this design extending beyond this. Whether multiple SRNs are allowed or not doesn't necessarily depend on whether its `plioUse` or not, so the two checks should be separate.  Since this variable is *only* checking if multiple SRNs shoudl be used or not, might as well rename to `multipleSRNsAllowed"

The auth should get all these things as params.

A better design might be to have separate components for a single-select SRN view vs a multi-SRN view, both of which will use the same SRN input component.

Reply:
I did initially do that; created two components: SingleSRNEntry, SRNEntry. But then I realized that a lot of code was repeating, the only thing that is different is, if it allows multiple entries, the var is of an object type, otherwise, it's a string. The plio code takes the `unique_id` as a string, so that was when I decided to just have one component and toggle between two variables.

Reply:
But yes, instead of hardcoding the `plioUse` var, I could check if the `Purpose` is `attendance` and the `subPurpose` is `plio`, then use one variable over the other.